packages/visual-dev/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from 'styled-components'
+import * as SVGs from './SVGs'
+
+type IconProps = OptionProps & StyleProps
+
+interface OptionProps {
+	icon: keyof typeof SVGs
+}
+
+interface StyleProps {
+	color?: string
+	size?: "small" | "medium" | "large" | string
```

Comment:
`size` could have it's own explicit type, e.g. `size?: IconSize` then somewhere else have `type IconSize = "small" | "medium" | "large" | string`. This improves maintainability a tiny bit.